apps/angular/src/app/features/home/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,41 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatTableModule } from '@angular/material/table';
+import { Observable } from 'rxjs';
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+import { Pagination } from '@js-camp/core/models/pagination';
+import { Anime } from '@js-camp/core/models/anime';
+import { NoEmptyPipe } from '@js-camp/angular/core/pipes/no-empty.pipe';
+
+/** Anime Table Component. */
+@Component({
+	selector: 'camp-anime-table',
+	standalone: true,
+	imports: [CommonModule, MatTableModule, NoEmptyPipe],
+	templateUrl: './anime-table.component.html',
+	styleUrl: './anime-table.component.css',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AnimeTableComponent {
+	/** Anime response observable. */
```

Comment:
```suggestion
	/** Anime page observable. */
```
Try not to use `response` for UI layer, since it's usually related to http calls